src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -120,16 +128,24 @@ class AssemblerConnectionManager(
         }
     }
 
+    fun cacheLoader(): CacheLoader<String,HikariDataSource> {
+        return CacheLoader.from { dbName ->
+            createDataSource(dbName!!, assemblerConfiguration.server.clone() as Properties, assemblerConfiguration.ssl)
+        }
+    }
+
     fun connect(dbName: String): HikariDataSource {
-        return connect(dbName, assemblerConfiguration.server.clone() as Properties, assemblerConfiguration.ssl)
+        return perDbCache.get(dbName)
     }
 
+    /* TODO: use #perDbCache or delete.
+     */
     fun connect(dbName: String, account: MaterializedViewAccount): HikariDataSource {
```

Comment:
I think this can be as deprecated, unless we want to use `AssemblyAnalyzationController` for tamatoa. @jokedurnez should we maintain that service or not? (querying aggregates, calculations... from materialized views)

Reply:
👍 for `@Deprecated`